srv/isc-agent/settings.py

```diff
@@ -57,6 +57,10 @@
 )
 DATABASE_SESSION = Session(DATABASE_ENGINE)
 
+# CSS certification key and certificate
```

Comment:
I dont think CSS is the write term here. CSS is more for frontend design stuff.

Reply:
Rookie mistake.  My bad.